service/game.ts

```diff
@@ -1,39 +1,38 @@
 import { match, P } from "ts-pattern";
 import { read, Move } from "../model/move";
-import { lastGame, allGames } from "../sql/db";
-import { Result, resultArray, results } from "../model/result";
-import { logRes } from "../sql/db";
+import { ResultArray, resultArray, result } from "../model/result";
+import { logRes, allGames, lastGame } from "../sql/db";
 
 export function generateComputerMove(): Move {
   return read(String(Math.round(Math.random() * 2)));
 }
-export function playLogic(userMove: Move, computerMove: Move): string {
-  return match([userMove, computerMove])
+export async function playLogic(
+  userMove: Move,
+  computerMove: Move
+): Promise<string> {
+  const outcome: string[] = match([userMove, computerMove])
```

Comment:
suggerirei di fare destructuring tipo
```suggestion
  const [result, resultMessage] = match([userMove, computerMove])
```
per evitare di usare `outcome[0]` e `outcome[1]` nel seguito, che mi sembrano meno leggibili

Reply:
Che gioia, non avevo proprio considerato che si potesse usare destructuring in TS